<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="label">
       <attribute name="title">
        <string>Címke</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Név:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="modeLabel">
          <property name="text">
           <string>Tulajdonság</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="modeWidget">
          <item>
           <property name="text">
            <string>Van ilyen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Minden ilyen</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Hozzáadás</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QListWidget" name="existinglabel_listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="topping">
       <attribute name="title">
        <string>Feltét</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="nameLabel_2">
          <property name="text">
           <string>Név:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameLineEdit_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelLabel">
          <property name="text">
           <string>Címke:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QListWidget" name="label_listWidget"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="addButton_2">
          <property name="text">
           <string>Hozzáadás</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QListWidget" name="existingtopping_listWidget"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="priceLabel">
          <property name="text">
           <string>Ár:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="priceLineEdit">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pizza">
       <attribute name="title">
        <string>Pizza</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="nameLabel_3">
          <property name="text">
           <string>Név:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameLineEdit_3"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="basepriceLabel">
          <property name="text">
           <string>Alapár:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="basepriceSpinBox">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="toppingLabel">
          <property name="text">
           <string>Feltétek:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QListWidget" name="topping_listWidget"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="addButton_3">
          <property name="text">
           <string>Hozzáadás</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QListWidget" name="existingpizza_listWidget"/>
        </item>
        <item row="3" column="1">
         <widget class="QListWidget" name="filter_listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="basics">
       <attribute name="title">
        <string>Alapadatok</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="shopnameLabel">
          <property name="text">
           <string>Pizzéria neve:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="shopnameLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="transportcostLabel">
          <property name="text">
           <string>Szállítási költség:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="transportcostspinBox">
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Szerda</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Vasárnap</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTimeEdit" name="tStart"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Hétfő</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Kedd</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTimeEdit" name="mStart"/>
          </item>
          <item row="4" column="1">
           <widget class="QTimeEdit" name="fStart"/>
          </item>
          <item row="6" column="1">
           <widget class="QTimeEdit" name="sunStart"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Péntek</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Csütörtök</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Szombat</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTimeEdit" name="thStart"/>
          </item>
          <item row="5" column="1">
           <widget class="QTimeEdit" name="satStart"/>
          </item>
          <item row="2" column="1">
           <widget class="QTimeEdit" name="wStart"/>
          </item>
          <item row="0" column="2">
           <widget class="QTimeEdit" name="mEnd"/>
          </item>
          <item row="1" column="2">
           <widget class="QTimeEdit" name="tEnd"/>
          </item>
          <item row="2" column="2">
           <widget class="QTimeEdit" name="wEnd"/>
          </item>
          <item row="3" column="2">
           <widget class="QTimeEdit" name="thEnd"/>
          </item>
          <item row="4" column="2">
           <widget class="QTimeEdit" name="fEnd"/>
          </item>
          <item row="5" column="2">
           <widget class="QTimeEdit" name="satEnd"/>
          </item>
          <item row="6" column="2">
           <widget class="QTimeEdit" name="sunEnd"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="save_pushButton">
          <property name="text">
           <string>Mentés</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_jl">
    <property name="title">
     <string>Fájl</string>
    </property>
    <addaction name="actionBet_lt_s"/>
    <addaction name="actionKiment_s"/>
    <addaction name="separator"/>
    <addaction name="actionKil_p_s"/>
   </widget>
   <widget class="QMenu" name="menuWeboldal">
    <property name="title">
     <string>Weboldal</string>
    </property>
    <addaction name="actionGener_l_s"/>
   </widget>
   <addaction name="menuF_jl"/>
   <addaction name="menuWeboldal"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBet_lt_s">
   <property name="text">
    <string>Betöltés</string>
   </property>
  </action>
  <action name="actionKiment_s">
   <property name="text">
    <string>Kimentés</string>
   </property>
  </action>
  <action name="actionKil_p_s">
   <property name="text">
    <string>Kilépés</string>
   </property>
  </action>
  <action name="actionGener_l_s">
   <property name="text">
    <string>Generálás</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
